code/x_coverage_prisons.R

```diff
@@ -24,32 +24,39 @@ library(lubridate)
 library(janitor)
 library(openxlsx)
 library(tidylog)
-
+library(phsaaa)
 
 rm(list = ls())
 gc()
 
 
 source(here::here("code/0_housekeeping.R"))
 
-rm(cutoff_date, hb_list, financial_year_due, hist_path, simd_path,
-   year1, year1_end, year1_start, year2, year2_end, year2_start)
+rm(cutoff_date, hb_list, hist_path, simd_path,
+   year1, year1_start, year2, year2_end, year2_start)
 
 financial_year <- kpi_report_years[3]
 
-cutoff_date <- "31-03-1957"
+cutoff_date <- year1_end ## AMc note: this date not verified, was 31-03-1957 when came to do 202409 autumn run
```

Comment:
Come back to this. Check SPSS code and define this variable (in notes in script). Someone born 31Mar1958 is 66 today (28Oct2024).